src/clr-angular/data/stack-view/stack-block.ts

```diff
@@ -41,7 +48,9 @@ import { UNIQUE_ID, UNIQUE_ID_PROVIDER } from '../../utils/id-generator/id-gener
   // Custom elements are inline by default
   styles: [
     `
-        :host { display: block; }
+      :host {
```

Comment:
Should this be in a scss file? I don't think we use the styles array by convention.

Reply:
This was already there, its just reformatted.